### ISSUE TITLE
There is absolutely no way to unmime people who are mimed by the wizard

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#20286]
(WEB REPORT BY: mask2697 REMOTE: 172.93.109.202:7777)
# Revision
0732f797e3e5915fcf820e1765b84c813351a755

# Description

Being physically gibbed and having my head attached to a different body did not even save me, much less not having a damn mask anymore. 
# Steps to Reproduce

Get Mimed
# What you Expected

To be able to fucking undo it with effort
# What Actually Happened

THE MIME RIDE NEVER ENDS

**Comment by DamianX** [comment]
Intended feature #19013
could be stupid

**Comment by SGuivenne** [comment]
To compare.  With Horsehead you need to just splash people with poly acid (or poly smoke if you are a gambling man).

With the clowns curse and the elf curse you need significantly more poly acid, but its also not really worth it because its just annoying.  

With the mime curse you can now make invisible walls but you can no longer talk.   This is SIGNIFICANTLY worse than the others (with exception of horsehead which can be fixed with poly acid). 

All of these are also instantanious and cause you to drop your spaghetti which means you can have your ID snatched by the adjacent wizard while you are blinded by the spell effect.  

Simply put, if you are horseheaded you can at least bray/moo/oink/whinee incoherently into comms and pray.  With the mime curse you have no mouth and must scream as the wizard has his way with you somewhere.  This also destroys the crews ability to coordinate.

**Comment by DamianX** [comment]
He said "French curse becomes much more difficult to remove", I wonder how one is supposed to remove it now.

**Comment by Dacendeth** [comment]
Doesn't cloning still remove the wizard version?

**Comment by SGuivenne** [comment]
nope.

**Comment by Allspice** [comment]
Nice desu

**Comment by Dacendeth** [comment]
I suggest honk serum as a counter for miming out of curse then, or to make cloning fix it. Just make sure real mimes aren't affected.

**Comment by SGuivenne** [comment]
So we can all agree that infinite mimeage is a bit shit then right?
